src/test/javascript/spec/component/programming-exercise/programming-exercise-instructor-submission-state.component.spec.ts

```diff
@@ -163,17 +168,17 @@ describe('ProgrammingExerciseInstructorSubmissionStateComponent', () => {
 
         fixture.detectChanges();
 
-        expect(getExerciseSubmissionStateStub).to.have.been.calledOnceWithExactly(exercise.id);
+        expect(getExerciseSubmissionStateStub).toHaveBeenCalledWith(exercise.id);
```

Comment:
When converting this, also check toHaveBeenCalledTime(1), since toHaveBeenCalledWith would not detect it was wrongfully called multiple times.

Reply:
Didn't know that, thanks